test/Microsoft.Framework.Caching.Memory.Tests/EntryLinkExpirationTests.cs

```diff
@@ -238,51 +188,33 @@ public void NestedLinkContextsCanAggregate()
             string key2 = "myKey2";
             string key3 = "myKey3";
 
-            var link1 = new EntryLink();
-            var link2 = new EntryLink();
-
             var trigger2 = new TestTrigger() { ActiveExpirationCallbacks = true };
             var trigger3 = new TestTrigger() { ActiveExpirationCallbacks = true };
 
-            cache.GetOrSet(key1, context1 =>
+            IEntryLink link1 = null;
```

Comment:
they are being referenced outside the using block for verification of triggers etc.